src/objectTranslations/Contact-es.objectTranslation

```diff
@@ -1,129 +1,138 @@
+        <help>Cuando se selecciona, NPSP excluye este contacto del nombre del hogar (establecido en Configuración de NPSP | Personas | Hogares).</help>
+        <label>Excluir del nombre de hogar</label>
         <name>Exclude_from_Household_Name__c</name>
     </fields>
     <fields>
-        <help>El monto del primer crédito blando</help>
-        <label>Monto del primer crédito blando</label>
+        <help>El importe del primer crédito blando de este contacto.</help>
+        <label>Importe del primer crédito blando</label>
         <name>First_Soft_Credit_Amount__c</name>
     </fields>
     <fields>
-        <help>Fecha del primer crédito blando</help>
+        <help>La fecha del primer crédito blando de este contacto.</help>
         <label>Fecha del primer crédito blando</label>
         <name>First_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>Contiene el ID de la familia, ya sea que se usen cuentas o el objeto familias.</help>
-        <label>Id de familia</label>
+        <help>Contiene el Id. del registro del hogar (ya sea la cuenta del hogar o el objeto Hogar) con el que este contacto está asociado en estos momentos (solo lectura).</help>
+        <label>Id. de Hogar</label>
         <name>HHId__c</name>
     </fields>
     <fields>
-        <help>El monto del mayor crédito blando</help>
-        <label>Mayor monto de crédito blando</label>
+        <help>El importe más gran del crédito blando de este contacto.</help>
+        <label>Importe más grande de crédito blando</label>
         <name>Largest_Soft_Credit_Amount__c</name>
     </fields>
     <fields>
-        <help>Fecha del mayor crédito blando</help>
-        <label>Fecha del mayor crédito blando</label>
+        <help>La fecha del crédito blando más grande de este contacto.</help>
+        <label>Fecha del crédito blando más grande</label>
         <name>Largest_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>Monto el último crédito blando</help>
-        <label>Monto el último crédito blando</label>
+        <help>El importe del último crédito blando de este contacto.</help>
+        <label>Importe del último crédito blando</label>
         <name>Last_Soft_Credit_Amount__c</name>
     </fields>
     <fields>
-        <help>Fecha del último crédito blando</help>
+        <help>La fecha del último crédito blando de este contacto.</help>
         <label>Fecha del último crédito blando</label>
         <name>Last_Soft_Credit_Date__c</name>
     </fields>
     <fields>
-        <help>La cantidad de créditos blandos, cerradas en los últimos días definidos en la configuración del NPSP</help>
-        <label>Cantidad de créd. blandos últimos N días</label>
+        <help>El número de créditos blandos atribuidos a este contacto que se cerraron en los últimos N días. El valor de N se define en Configuración de NPSP | Donaciones | Datos estadísticos de donantes.</help>
+        <label>Número créd. blandos en últimos N días</label>
```

Comment:
@force2b To be consistent with the translations in other similar labels in this UI, I need to fix this to 
<label>Número créditos blandos en últimos N días</label>
What should I do to fix this or can you fix it in this PR?

Reply:
If you're just changing the one label, you can simply edit in this PR. 

However I want to be sure this is the **final** PR that we're going to use for Cumulus. This PR is based on the original `feature/ngohana/2019.07.localization` that I created back in July from your first PR, and it has all of the new labels and fixes that we made back then to address the identified localization issues. If you're comfortable **this** is the PR we're going with (and will merge), you can use the Edit button on the `Contact-es.objectTranslation` file to change it (or I can make the label change if you prefer). Either way is fine as long as you're comfortable with PR.